### PR TITLE
monitoring: trigger 'gather-build-stats' job

### PR DESCRIPTION
**Pull request by sodar**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -191,6 +191,11 @@ pipeline {
                         deleteDir()
                     }
                 }
+
+                build job: 'WinContrail/gather-build-stats',
```
Comment (jablonskim) [inline Jenkinsfile]: Please add `wait: false`.
There will be a race condition. It should be fixed in new PR.

**Review by jablonskim — CHANGES_REQUESTED** [review]
MJ & JI

**Review by jacek-izykowski — CHANGES_REQUESTED** [review]
gather-build-stats always receives "IN PROGRESS".

**Review by jacek-izykowski — DISMISSED** [review]
There's race condition now, but as we agreed - that's going to be handled in next PR.

**Review by jablonskim — APPROVED** [review]

**Review by jacek-izykowski — APPROVED** [review]

**Pull request merged**